feature/interests/src/main/java/com/google/samples/apps/nowinandroid/feature/interests/InterestsItem.kt

```diff
@@ -53,61 +52,43 @@ fun InterestsItem(
-        modifier = modifier,
-    ) {
-        Row(
-            verticalAlignment = Alignment.CenterVertically,
-            modifier = Modifier
-                .weight(1f)
-                .clickable { onClick() }
-                .padding(vertical = itemSeparation),
-        ) {
+    ListItem(
+        leadingContent = {
             InterestsIcon(topicImageUrl, iconModifier.size(64.dp))
-            Spacer(modifier = Modifier.width(24.dp))
-            InterestContent(name, description)
-        }
-        NiaIconToggleButton(
-            checked = following,
-            onCheckedChange = onFollowButtonClick,
-            icon = {
-                Icon(
-                    imageVector = NiaIcons.Add,
-                    contentDescription = stringResource(
-                        id = string.card_follow_button_content_desc,
-                    ),
-                )
-            },
-            checkedIcon = {
-                Icon(
-                    imageVector = NiaIcons.Check,
-                    contentDescription = stringResource(
-                        id = string.card_unfollow_button_content_desc,
-                    ),
-                )
-            },
-        )
-    }
-}
-
-@Composable
-private fun InterestContent(name: String, description: String, modifier: Modifier = Modifier) {
-    Column(modifier) {
-        Text(
-            text = name,
-            style = MaterialTheme.typography.headlineSmall,
-            modifier = Modifier.padding(
-                vertical = if (description.isEmpty()) 0.dp else 4.dp,
-            ),
-        )
-        if (description.isNotEmpty()) {
-            Text(
-                text = description,
-                style = MaterialTheme.typography.bodyMedium,
+        },
+        headlineText = {
+            Text(text = name)
+        },
+        supportingText = {
+            Text(text = description)
+        },
+        trailingContent = {
+            NiaIconToggleButton(
+                checked = following,
+                onCheckedChange = onFollowButtonClick,
+                icon = {
+                    Icon(
+                        imageVector = NiaIcons.Add,
+                        contentDescription = stringResource(
+                            id = string.card_follow_button_content_desc,
+                        ),
+                    )
+                },
+                checkedIcon = {
+                    Icon(
+                        imageVector = NiaIcons.Check,
+                        contentDescription = stringResource(
+                            id = string.card_unfollow_button_content_desc,
+                        ),
+                    )
+                },
             )
-        }
-    }
+        },
+        modifier = modifier.clickable(onClick = onClick),
+        colors = ListItemDefaults.colors(
+            containerColor = if (isSelected) MaterialTheme.colorScheme.surface else Color.Transparent,
```

Comment:
In addition to changing the visual of the card when selected, can we also update the semantics for talkback?

Reply:
Can we use `selectableGroup` and `selectable` as well?

Reply:
Done!